spec/index.html

```diff
@@ -927,9 +915,9 @@ <h2>Annotation Syntax</h2>
         triples and/or <a data-cite="RDF12-CONCEPTS#dfn-triple-term">triple terms</a>.
         As with a <a>reified triple</a>,
         the annotation syntax allows the definition of
-        one or more <a>reifiers</a> as either an <a data-cite="RDF12-CONCEPTS#dfn-iri">IRI</a>
-        or <a data-cite="RDF12-CONCEPTS#dfn-blank-node">blank node</a>
-        preceded by <a href="#cp-tilda"><code>~</code></a>,
+        one or more <a>reifiers</a> as either an <a data-cite="RDF12-CONCEPTS#dfn-iri">IRIs</a>
+        or <a data-cite="RDF12-CONCEPTS#dfn-blank-node">blank nodes</a>
+        each preceded by <a href="#cp-tilda"><code>~</code></a>,
```

Comment:
Note, "[tilde](https://en.wikipedia.org/wiki/Tilde)" `~` is not spelled "[tilda](https://en.wikipedia.org/wiki/Tilda)". That's probably worth a global search & replace.
```suggestion
        one or more <a>reifiers</a> as either <a data-cite="RDF12-CONCEPTS#dfn-iri">IRIs</a>
        or <a data-cite="RDF12-CONCEPTS#dfn-blank-node">blank nodes</a>,
        each preceded by a tilde (<a href="#cp-tilda"><code>~</code></a>),
```